website/content/en/status/report-2025-04-2025-06/intro.adoc

```diff
@@ -3,12 +3,12 @@ Here is the second 2025 status report, with 32 entries.
 As for the preceding quarters, this report is published just a few days before calls for 2025Q3 report submissions are sent.
 Indeed, although according to our link:https://docs.freebsd.org/en/articles/freebsd-status-report-process/#_timeline[timeline] we should have published this report in July (general rule is publication should happen within the month just after the calls for reports are sent), we kept receiving important reports until the end of August.
 This is both a positive and a negative thing.
-On a hand, it means that our FreeBSD community is busy fixing existing issues and implementing new features, making the OS we love better and better every day; it means that the community works so intensely that very little time remains for reporting.
+On one hand, it means that our FreeBSD community is busy fixing existing issues and implementing new features, making the OS we love better and better every day; it means that the community works so intensely that very little time remains for reporting.
 On the other hand, it means that news in these reports is always two months old when published.
 Two months is not bad, especially if we consider that FreeBSD communication happens on many other channels too, but it would be nice if we could improve it.
 
 If you are a late submitter, please take some time to evaluate if there is anything you can do to improve your report submission punctuality.
-The Status team is always glad to ease the submission process: if there is something we can do for you, just ask.
+The Status Team is always glad to ease the submission process: if there is something we can do for you, just ask.
 If you are a contributor or just a FreeBSD user, please consider contributing more if you can.
```

Comment:
```suggestion
If you are a contributor or just a FreeBSD user, please consider contributing more, if you can.
```

Reply:
@lsalvadore hi, at https://github.com/freebsd/freebsd-doc/blob/main/website/content/en/status/report-2025-04-2025-06/intro.adoc the additional comma is in the wrong place (should be to the right, not left, of "more"). 

I doubt that anyone other than me will notice :-)

Reply:
Oh, sorry about that! I am going to fix it in some other commit with other fixes coming. Thanks!